### PR TITLE
filter out claimed issues from github response

### PR DESCRIPTION
**Pull request by brandonmbanks**
closes https://github.com/gatsbyjs/store.gatsbyjs.org/issues/155

### CODE REVIEW

File: src/components/Dashboard/OpenIssues.js [hunk 1]
```diff
@@ -36,7 +41,7 @@ export default class OpenIssues extends React.Component {
     ) : (
       <>
         <Subheading>Issues We Could Use Your Help With</Subheading>
-        <IssueList issues={issues.slice(0, 3)} />
+        <IssueList issues={this.state.issues.slice(0, 3)} />
```
Comment (brandonmbanks) [inline src/components/Dashboard/OpenIssues.js]: Had to update this to not use the components issues variable. By using the components state we can actually get the filtered list.

**Comment by brandonmbanks** [comment]
Closes https://github.com/gatsbyjs/store.gatsbyjs.org/issues/155

**Comment by brandonmbanks** [comment]
Looking at GitHub’s API docs it doesn’t look like there is a way to exclude labels in the request. So I’m getting the response and then filtering the claimed issues out.

File: src/components/Dashboard/OpenIssues.js [hunk 2]
```diff
@@ -26,7 +26,12 @@ export default class OpenIssues extends React.Component {
 
   componentDidMount() {
     getIssues().then(issues => {
-      this.setState({ issues });
+      const unclaimedIssues = issues.filter(
+        issue => issue.labels.filter(
```
Comment (hackily) [inline src/components/Dashboard/OpenIssues.js]: Using a second nested filter is a little bit verbose, might I suggest something like this?
```
const unclaimedIssues = issues.filter(issue => !issue.labels.includes('Hacktoberfest - Claimed'))
```

Comment (brandonmbanks) [reply src/components/Dashboard/OpenIssues.js]: The problem is that `labels` is a list of `Issue` objects, not just a list of of strings. So I have to access the name property of each label. I could map the labels to build up an array of strings.

Here is a example issue response.
```
{
  "id": 779050111,
  ...,
  "labels": [
    {
      "id": 214544675,
      "node_id": "MDU6TGFiZWwyMTQ1NDQ2NzU=",
      "url": "https://api.github.com/repos/gatsbyjs/gatsby/labels/status:%20help%20wanted",
      "name": "status: help wanted",
      "color": "0e8a16",
      "default": false
    },
    {
      "id": 779050110,
      "node_id": "MDU6TGFiZWw3NzkwNTAxMTA=",
      "url": "https://api.github.com/repos/gatsbyjs/gatsby/labels/type:%20feature%20or%20enhancement",
      "name": "type: feature or enhancement",
      "color": "c5def5",
      "default": false
    }
  ],
}
```

But good idea, I think that is a little cleaner using `includes` with `map` instead. It removes the need for the length check. I'll push that up. Thanks @hackily!

**Comment by brandonmbanks** [comment]
Glad to help! Happy to contribute to a great product.

**Comment by ItsPugle** [comment]
You're a star 🌟

**Pull request merged**